twake/backend/node/src/core/platform/services/database/services/orm/connectors/cassandra/typeTransforms.ts

```diff
@@ -69,7 +73,7 @@ export const transformValueToDbString = (
     return "''"; //Not implemented yet
   }
   if (type === "string" || type === "json") {
-    if (type === "json") {
+    if (type === "json" && v !== null) {
```

Comment:
Why is it needed?
If you do you must at least also do it in mongo connector.

Reply:
Already done 😄